.github/workflows/dotnetcore.yml

```diff
@@ -0,0 +1,31 @@
+name: .NET Core
+
+on: [push]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Chectout
+      uses: actions/checkout@v1
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.0.100
+    - name: Build with dotnet
+      run: dotnet build --configuration Release
```

Comment:
Shouldn't we be generating nuget packages via the `pack` command and pushing them to a CI package registry?

Reply:
I think so too. However, the difficulty I have now is that I don't know how to make the version number increase automatically.

Reply:
Blockchain Toolkit projects use [NerdBank GitVersioning](https://github.com/AArnott/Nerdbank.GitVersioning). Highly recommended

Reply:
I had in the oast the same problem, you can see here my result

https://github.com/Red4Sec/TuringMachine/blob/d935cd5fa30488b40d747774c011a45c9e81329f/src/TuringMachine.Core/TuringMachine.Core.csproj#L52

Reply:
I'm not a fan of an build versioning scheme that depends on the time of day the build is generated

Reply:
https://github.com/neo-project/neo-vm/packages

Done.